.eslintrc

```diff
@@ -5,11 +5,28 @@
   ],
   "rules": {
     "curly": ["error", "multi-line"],
-    "prettier/prettier": ["error", { "trailingComma": "es5" }],
+    "prettier/prettier": "error",
     "no-case-declarations": "error"
```

Comment:
should we just move these rules into the babel config? If we wanted we could even move the config into this repo

Reply:
Can do that next.